### PR TITLE
Lines should extend to bottom of content

### LINKED ISSUE #1101: front end - lines should extend to bottom of content
The vertical line dividing space on the patient screens should extend to the bottom of the content. 

screen shot 2017-06-05 at 7 21 32 pm

### PR DESCRIPTION
**Pull request by colinmcglynn**
I rule and have completed some work on Case Manager that's ready for review!

This pull request makes the following changes:
* Moves the border from the left `#menu` to the right side `#sections`
* Adds padding to `#sections` as a percent so that is responsive
* Removes the `#menu` div and column that was being used to create padding in the menu. 

It relates to the following issue #s: 
* Fixes #1101 https://github.com/DCAFEngineering/dcaf_case_management/issues/1101

screen shot 2017-07-03 at 12 24 04 am

### CODE REVIEW

**Comment by colinmcglynn** [comment]
I'm not a super experienced designer/frontender, so if you want me to tackle this a different way, let me know.  I'll be happy to!

**Comment by colinxfleming** [comment]
My dude 👍 

taking a look at this now!

File: app/views/patients/edit.html.erb [hunk 1]
```diff
@@ -8,13 +8,11 @@
 </div>
 
 <div id="patient_detail" class="row">
-  <div class="col-sm-4">
-    <div id="menu" class="col-sm-9">
-      <%= render 'patients/menu', patient: @patient %>
-    </div>
+  <div class="col-sm-3">
```
Comment (colinxfleming) [inline app/views/patients/edit.html.erb]: Can you do me a solid and revert these changes? We want the header that shows up in sections to line up with the LMP at intake header, and this breaks the grid. (That, and we have some javascript that requires the `menu` id; once this change goes in the tests should all pass.)

File: app/assets/stylesheets/pregnancies.scss [hunk 2]
```diff
@@ -16,8 +16,12 @@ $font-size: 17px;
   }
 }
 
-#menu {
-  border-right: 1px solid black;
+#sections {
```
Comment (colinxfleming) [inline app/assets/stylesheets/pregnancies.scss]: nailed it!

File: app/assets/stylesheets/pregnancies.scss [hunk 3]
```diff
@@ -16,8 +16,12 @@ $font-size: 17px;
...
+  border-left: 1px solid black;
+  padding-left: 15%;
```
Comment (colinxfleming) [inline app/assets/stylesheets/pregnancies.scss]: Can you pop out this padding-left? I don't want to remix this too heavy without getting approval from team ux, and I know they're in the process of undertaking a review of everything.

File: app/assets/stylesheets/pregnancies.scss [hunk 4]
```diff
@@ -16,8 +16,12 @@ $font-size: 17px;
...
+}
+
+#patient_detail {
```
Comment (colinxfleming) [inline app/assets/stylesheets/pregnancies.scss]: I think we want to change this id to menu, right?

File: app/views/patients/edit.html.erb (hunk 1 above)
Comment (colinmcglynn) [inline app/views/patients/edit.html.erb]: sure thing!

File: app/assets/stylesheets/pregnancies.scss (hunk 4 above)
Comment (colinmcglynn) [inline app/assets/stylesheets/pregnancies.scss]: Yeah, since I'm adding the `#menu` div back in, I will change this back to `#menu` as well

File: app/assets/stylesheets/pregnancies.scss (hunk 3 above)
Comment (colinmcglynn) [inline app/assets/stylesheets/pregnancies.scss]: I was trying to maintain the white space that you had here, here's what it looks like with no padding.
screen shot 2017-07-03 at 10 35 55 am If that works for you, I'll push it

Comment (colinxfleming) [inline app/assets/stylesheets/pregnancies.scss]: Ah, I see what you mean - my mistake, I thought this was what was breaking grid! If the Patient Information header lines up with LMP at intake, I'm good with it.

**Comment by colinmcglynn** [comment]
Okay second try, I tweaked the padding to keep the grid alligned 

Here are a couple of different widhts 
screen shot 2017-07-03 at 11 40 13 pm

screen shot 2017-07-03 at 11 40 49 pm

**Comment by colinxfleming** [comment]
Looks good to me! Thanks a ton @colinmcglynn !!

**Comment by lwaldsc** [comment]
Just popping in to say thank you, @colinmcglynn! Welcome to the team!

**Comment by colinmcglynn** [comment]
Thanks, glad to be here!
